voxblox/include/voxblox/core/layer.h

```diff
@@ -190,12 +190,12 @@ class Layer {
     }
   }
 
-  void getAllUpdatedBlocks(BlockIndexList* blocks) const {
+  void getAllUpdatedBlocks(Update::State bit, BlockIndexList* blocks) const {
```

Comment:
nit: not sure this is self-explanatory, maybe consider the renaming suggestion above, I think `void getAllUpdatedBlocksconst Update::Flag flag, ...)` would be more intuitive.